packages/internals/src/cli/getSchema.ts

```diff
@@ -246,7 +246,6 @@ export async function getSchemaDir(schemaPathFromArgs?: string): Promise<string
   return path.dirname(schemaPath)
 }
 
-// TODO: This should probably return string | null to stay consistent with the other functions
```

Comment:
This comment was removed because we looked into it with @millsp and @SevInf and concluded that it makes more sense as it currently is.